src/components/http/_tests/HttpService_fetch.test.brs

```diff
@@ -2,47 +2,71 @@ function TestSuite__HttpService_fetch() as Object
   ts = HttpServiceTestSuite()
   ts.name = "HttpService_fetch"
 
-  ts.addTest("it sends request with given params", function (ts as Object) as String
+  it("sends request with given params", function (_ts)
```

Comment:
I intentionally left it without type - we don't use _ts, so such information is pointless and is just a pain in the neck. Test function sometimes returns a string, sometimes an object. It's an "end function", so we don't gain anything from determining the type and we can't even use a single template

Reply:
For some test functions, you did use types.
Overall I agree, but this could be mentioned somewhere instead of just not doing it, as we have it defined for all older tests.